coordinator/rec_util.py

```diff
@@ -110,11 +113,35 @@ def record(r, array, instances):
 
     return set(instances)
 
-def add_unprocessed(r, recording, datadir):
+def check_primary_time(r, array):
+    """Check if the current recording is primary time.
+    """
+    array_num = array[-1] # last char is array number
+    key = f"{array}:subarray_{array_num}_script_proposal_id"
+    proposal_id = r.get(key)
+    if not proposal_id:
+        return
+    log.info(f"Retrieved currrent proposal ID: {proposal_id}")
+    if proposal_id.strip("'") == "DDT-20230920-DC-01":
```

Comment:
is this some special case? you might want to add a comment because I bet you look back at this part in a few months and think, what was this

Reply:
Added a comment to help remembering. Thanks for taking a look at all this!